CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Next Release
 -------------
 
+1.5.2
```

Comment:
NIT: Don't put version in changelog because you don't know what other changes will go in the release that could change the version number.